aptos/contract/aux/sources/clob_market.move

```diff
@@ -901,6 +903,8 @@ module aux::clob_market {
         let base_decimals = coin::decimals<B>();
         // This invariant ensures that the smallest possible trade value is representable with quote asset decimals
         assert!((lot_size as u128) * (tick_size as u128) / exp(10, (base_decimals as u128)) > 0, E_INVALID_TICK_OR_LOT_SIZE);
+        // This invariant ensures that the smallest possible trade value has no rounding issue with quote asset decimals
```

Comment:
nit: but would be slight gas savings to store the denominator in a variable to avoid double `exp`.

Reply:
good point. updated.